lib/helpers/client_schema.js

```diff
@@ -486,9 +486,11 @@ module.exports = function getSchema(provider) {
 
         switch (this.application_type) { // eslint-disable-line default-case
           case 'web': {
-            if (!['https:', 'http:'].includes(protocol)) {
-              invalidate('redirect_uris must only contain web uris');
-            }
+            // Disable protocol validation on web clients.
```

Comment:
@ChadKillingsworth would be nice to add this link as a reference in the code as to why we're doing this: https://github.com/panva/node-oidc-provider/issues/695